hw/top_earlgrey/data/chip_testplan.hjson

```diff
@@ -900,6 +901,173 @@
       tests: []
     }
 
+    // SYSRST_CTRL (pre-verified IP) integration tests:
+    {
+      name: chip_sysrst_ctrl_inputs
+      desc: '''Verify that the sysrst_ctrl input pin values can be read.
+
+            - Drive a known value on ac_reset, ec_rst_l, pwrb and key* pins at the chip IOs.
+            - Read the pin_in_value CSR to check for correctness.
+            '''
+      milestone: V2
+      tests: []
+    }
+    {
+      name: chip_sysrst_ctrl_outputs
+      desc: '''Verify that the sysrst_ctrl output pin values can be set.
+
+            - Drive a known value on ac_reset, ec_rst_l, pwrb and key* pins at the chip IO inputs.
+            - Verify that sysrst ctrl correctly loops them back to the chip IO outputs.
+            - Write the pin allowed ctl register to allow some of the pins to be overridden with
```

Comment:
Consistently use ctrl
```suggestion
            - Write the pin allowed ctrl register to allow some of the pins to be overridden with
```

Reply:
Aah, I mimicked the design which uses `ctl` in the CSR name.